lib/profileAgents.js

```diff
@@ -411,15 +415,15 @@ exports.delegateCapabilityInvocationKey = async ({
   const key = await keystoreAgent.getAsymmetricKey(capabilityInvocationKey);
 
   const request = {
-    referenceId: 'profileAgentCapabilityInvocationKey',
     // string should match KMS ops
     allowedAction: 'sign',
-    invoker,
-    invocationTarget: {
-      id: capabilityInvocationKey.kmsId,
-      type: capabilityInvocationKey.type,
-      publicAlias: key.id
-    },
+    controller: invoker,
```

Comment:
Ideally, we pass in `controller` and don't use `invoker` anywhere anymore. I can do this refactor later if need be, but it should happen before release of this major version.